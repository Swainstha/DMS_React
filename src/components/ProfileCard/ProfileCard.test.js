import testSnapFunction from '../../util/testSnapFunction';
import ProfileCard from './index'
import React from 'react';


testSnapFunction("<ProfileCard />", "Snapshot test for ProfileCard ",<ProfileCard /> )