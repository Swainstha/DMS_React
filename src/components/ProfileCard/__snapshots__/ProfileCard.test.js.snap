// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileCard /> Snapshot test for ProfileCard  1`] = `ShallowWrapper {}`;
